server/index.js

```diff
@@ -21,16 +21,13 @@ app.get("/api/kanji/", async (req, res) => {
   const katakanaArr = katakana.split("");
 
   const generatedKanjiObj = await katakanaToKanji(katakanaArr);
-  console.log(generatedKanjiObj);
-
   const generatedDefinitionObj = await getKanjiDefinitions(generatedKanjiObj);
-  console.log(generatedDefinitionObj);
 
   const kanjiNames = combiner(generatedKanjiObj);
   const kanjiDefinitions = combiner(generatedDefinitionObj);
   const kanjiData = getKanjiData(hiraganaArr, kanjiNames, kanjiDefinitions);
 
-  res.status(200).send(JSON.stringify({ kanjiData }));
+  res.send(kanjiData);
```

Comment:
Can you change it to `res.status(200).send(kanjiData)` so that it sends data only when the HTTP request is successful?